doc-src/install.rst

```diff
@@ -31,287 +336,990 @@ is currently tested only on x86_64 and ARM.
+ownership.
 
-  $ echo "~sys-cluster/charliecloud-9999 \*\*" >> /etc/portage/package.accept_keywords
+To configure Buildah in rootless mode, make sure your config files are in
+:code:`~/.config/containers` and they are correct. Particularly if your system
+also has configuration in :code:`/etc/containers`, problems can be very hard
+to diagnose.
 
-openSUSE and SUSE
------------------
+.. For example, with different mistakes in
+   :code:`~/.config/containers/storage.conf` and
+   :code:`/etc/containers/storage.conf` present or absent, and all in rootless
+   mode, we have seen various combinations of:
 
-Charliecloud is included in openSUSE Tumbleweed.
-For SUSE Linux Enterprise users, it's available via
-`SUSE Package Hub
-<https://packagehub.suse.com/packages/charliecloud/>`_.
+     * error messages about configuration
+     * error messages about :code:`lchown`
+     * using :code:`storage.conf` from :code:`/etc/containers` instead of
+       :code:`~/.config/containers`
+     * using default config documented for rootless
+     * using default config documented for rootful
+     * exiting zero
+     * exiting non-zero
+     * completing the build
+     * not completing the build
 
-.. list-table::
-   :widths: auto
+   We assume this will be straightened out over time, but for the time being,
+   if you encounter strange problems with Buildah, check that your config
+   resides only in :code:`~/.config/containers` and is correct.
 
-   * - Package name
-     - :code:`charliecloud`, :code:`charliecloud-doc` and :code:`charliecloud-examples`
-   * - Maintainers
-     - Ana Guerrero Lopez (:code:`aguerrero@suse.com`)
-       and Christian Goll (:code:`cgoll@suse.com`)
-   * - Bug reports to
-     - `openSUSE Bugzilla <https://en.opensuse.org/openSUSE:Submitting_bug_reports>`_
-   * - Packaging source code
-     - `openSUSE Build Service <https://build.opensuse.org/package/show/network:cluster/charliecloud>`_
+Python package "lark-parser"
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+PyPI has two incompatible packages that provide the module :code:`lark`,
+"`lark-parser <https://pypi.org/project/lark-parser/>`_" and "lark". You want
+"lark-parser".
 
-RPM-based distributions
------------------------
+skopeo
+~~~~~~
 
-An RPM :code:`.spec` file is provided in the Charliecloud source code. We are
-actively seeking distribution packagers to adapt this into official packages!
+.. todo:: Do we have anything to say about installing `skopeo
+          <https://github.com/containers/skopeo>`_?
 
-.. list-table::
-   :widths: auto
+umoci
+~~~~~
 
-   * - Repositories
-     - none yet
-   * - Maintainer
-     - Oliver Freyermuth (:code:`o.freyermuth@googlemail.com`)
-   * - Bug reports to
-     - Charliecloud's GitHub issue tracker
-   * - Packaging source code
-     - in Charliecloud: :code:`packaging/redhat`
+.. todo:: Do we have anything to say about intsalling `umoci
+          <https://github.com/openSUSE/umoci>`_?
```

Comment:
Currently, we install umoci like this in travis-install and the WIP Vagrantfile. 

``` 
    wget https://github.com/openSUSE/umoci/releases/download/v0.4.4/umoci.amd64
    sudo chmod 755 umoci.amd64
    sudo mv umoci.amd64 /usr/bin/umoci
    umoci --version
``` 
We should either include those installation directions or check if the newest Umoci is sufficient.

Reply:
The static binary makes installing umoci painless. However, I was completely unaware that umoci provided a static binary despite visiting their release page on github several times. 

IMO, we should specify what options are available for installing it (rpm, static binary, go, etc.) and provide links.

Reply:
> IMO, we should specify what options are available for installing it (rpm, static binary, go, etc.) and provide links.

As a general rule, I worry about duplicating information better presented elsewhere, which would include install instructions for other software.

That said, I'm fine with filling gaps, e.g., the fact that there is a static build readily available.